stolostron/backlog#26798 search deprec (#4475)
open-cluster-management/rhacm-docs: release_notes/deprecate_remove.adoc

|===
|Product or category | Affected item | Version | Recommended action | More details and links

| Search

> | `SearchCustomization.open-cluster-management.io` custom resource definition is removed.

Reviewer: ```suggestion
| `SearchCustomizations.open-cluster-management.io` custom resource definition is removed.
```

| 2.7

> | Use custom resource definition `search.open-cluster-management.io/v1alpha1`. 

Reviewer: ```suggestion
| Use `search.open-cluster-management.io/v1alpha1` to customize search.
```

| 

| Search

> | PostgreSQL replaced RedisGraph as the internal database.

Reviewer: ```suggestion
| RedisGraph was replaced by PostgreSQL as the internal database.
```

> | Search
> | RedisGraph is replaced as the source database
> | 2.7
> | No change required. 
> | PostgreSQL replaced RedisGraph as the source database.

Reviewer: Would this be better?
```suggestion
| Search
| PostgreSQL replaced RedisGraph as the internal database
| 2.7
| No change required. 
| The search component was reimplemented using PostgreSQL as the internal database.
```

> | The search component was reimplemented using PostgreSQL as the internal database.

Reviewer 2: I recommend that you change "was" to "is"
